lib/extract_ttc.rb

```diff
@@ -7,16 +7,23 @@
 
```

Comment:
This PR handles some runtime dependency issue? Do we need to remove the "require" line?

Reply:
good catch those 2 lines above need to be removed